runtime/autoload/help.vim

```diff
@@ -0,0 +1,40 @@
+if exists("g:loaded_help")
+  finish
+endif
+let g:loaded_help = 1
+
+function! help#topic() abort
```

Comment:
Is this function obvious enough that it doesn't need a comment? I'm too green to judge. Would it qualify as obvious if it had tests?